panc/src/main/scripts/panlint

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+# coding=utf8
+
+import re
+import argparse
+from glob import glob
+from colorama import Fore, Style, init as colorama_init
+from sys import stdout, exit as sys_exit
+from inspect import getmembers, ismethod
+from prettytable import PrettyTable
+
+RS_COMMENT = r'(?:#.*|@{.*})$'
+
+RE_STRING = re.compile(r'''('.*?'|".*?"(?<!\\"))''')
+
+RE_FIRST_LINE = re.compile(r'^(?:(?:declaration|unique|structure|object) )?template (?:(?:[\w-]+/)+)?[\w-]+;$')
+RE_COMMENT_LINE = re.compile(r'^\s*' + RS_COMMENT)
+RE_TRAILING_COMMENT = re.compile(r'\s*' + RS_COMMENT)
+RE_ANNOTATION = re.compile(r'@{(?:.|\n)*?}')
```

Comment:
use `re.S` flag to allow `.` to match `\n`.
also use `@\w*\{` (and how does this work without escaped `{}`?)